lib/matplotlib/table.py

```diff
@@ -285,7 +285,7 @@ def __init__(self, ax, loc=None, bbox=None, **kwargs):
         loc : str
             The position of the cell with respect to *ax*. This must be one of
             the `~.Table.codes`.
-        bbox : `.Bbox` or None
+        bbox : `.Bbox` or [xmin, ymin, width, height] or None
```

Comment:
```suggestion
        bbox : `.Bbox` or [xmin, ymin, width, height], optional
```
For consistency with your change on the `table` function.

Reply:
Done!